### ISSUE TITLE
Inconsistent month capitalization in Italian with biblatex

### ISSUE THREAD

**Issue by marcopernpruner** [plk/biblatex#1367]
In Italian, month names are always written in lowercase unless they are the first word after a period. However, when using biblatex with the `italian` option in the `babel` package, there are inconsistencies in month capitalization.

Consider the following minimal working example (MWE):

```latex
\documentclass[11pt]{article}
\usepackage[italian]{babel}
\usepackage[backend=biber,dateabbrev=false]{biblatex}

\begin{filecontents*}{Bibliography.bib}
@inproceedings{Pub1,
    author = {Rossi, Mario and Verdi, Giovanni},
    title = {Title Example},
    booktitle = {Proceedings of the Text Conference},
    date = {2020-04}
}

@inproceedings{Pub2,
    author = {Rossi, Mario and Verdi, Giovanni},
    title = {Title Example},
    booktitle = {Proceedings of the Text Conference},
    publisher = {PublisherX},
    date = {2020-04}
}

@inproceedings{Pub3,
    author = {Rossi, Mario and Verdi, Giovanni},
    title = {Title Example},
    booktitle = {Proceedings of the Text Conference},
    publisher = {PublisherX},
    date = {2020-04-05}
}

@inproceedings{Pub4,
    author = {Rossi, Mario and Verdi, Giovanni},
    title = {Title Example},
    booktitle = {Proceedings of the Text Conference},
    date = {2020-04-05}
}
\end{filecontents*}

\addbibresource{Bibliography.bib}

\begin{document}
\nocite{*}
\printbibliography
\end{document}
```

The generated bibliography output is:

```
[1] Mario Rossi e Giovanni Verdi. "Title Example". In: Proceedings of the Text Conference. Aprile 2020.
[2] Mario Rossi e Giovanni Verdi. "Title Example". In: Proceedings of the Text Conference. PublisherX, aprile 2020.
[3] Mario Rossi e Giovanni Verdi. "Title Example". In: Proceedings of the Text Conference. PublisherX, 5 aprile 2020.
[4] Mario Rossi e Giovanni Verdi. "Title Example". In: Proceedings of the Text Conference. 5 Aprile 2020.
```

Analyzing the cases:
1. The month capitalization ("Aprile") is **correct**: it is the first word after a period, so it must be uppercase.
2. The month capitalization ("aprile") is **correct**: it is not the first word after a period, so it must be lowercase.
3. The month capitalization ("aprile") is **correct**: it is not the first word after a period, so it must be lowercase.
4. The month capitalization ("Aprile") is **incorrect**: it is **not** the first word after a period, so it should be lowercase.

In the last case, it appears that when the whole date follows a period, the month is always printed with an initial capital letter, regardless of its position. This results in inconsistencies in Italian bibliographies.

**Comment by moewew** [comment]
Thanks for reporting this. Interestingly, `italian.lbx` has

https://github.com/plk/biblatex/blob/289311628d1f71a4aed5bf516dbd5614a864a970/tex/latex/biblatex/lbx/italian.lbx#L16-L29

so it appears as though one of the original language contributors preferred this behaviour.

**Comment by moewew** [comment]
@matteocoder @matteofg @fargiolas @eg9 You're our regular Italian contributors. Any comments on this?

**Comment by matteocoder** [comment]
@moewew in Italian, the names of the months are treated as regular words, and thus are capitalised only after a period.

An interesting excerpt from [Treccani](https://www.treccani.it/enciclopedia/giorni-e-mesi-nomi-dei-prontuario_(Enciclopedia-dell'Italiano)/):
> The names of days and months are normally spelled with a lower case initial, but often, especially the names of months within dates, are found written with a capital initial because they are equated with proper names and due to the influence of English spelling.

**Comment by matteofg** [comment]
I have looked in several grammar textbook and "manuals of style" and found nothing about it.
From a strictly formal point of view, it's true that the month is not at the beginning of the period. However, it's the first word formed by letters, so I don't see any particular problem with capitalizing the month (imho).

**Comment by marcopernpruner** [comment]
> From a strictly formal point of view, it's true that the month is not at the beginning of the period. However, it's the first word formed by letters, so I don't see any particular problem with capitalizing the month (imho).

Italian rules require capitalization for words right after a period. I do not think it is correct to "postpone" capitalization to the second word just because the sentence starts with a number.
